app/models/user.rb

```diff
@@ -0,0 +1,38 @@
+require 'bcrypt'
+
+class User
+  include DataMapper::Resource
+
+  attr_reader :password
+  attr_accessor :password_confirmation
+
+  property :id, Serial
+  property :handle, String, required: true
+  property :email, String, required: true, unique: true
+  property :password_digest, Text
+
+  validates_presence_of :email
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.